com.dhsdevelopments.Climaxima.appdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This won't pass the appdata requirements https://github.com/flathub/flathub/wiki/App-Requirements#Appstream the guidelines page might help too https://github.com/flathub/flathub/wiki/AppData-Guidelines

Reply:
As best as I can tell, the only thing that s missing is the application icon. I will add this shortly.

Reply:
I ran the validator tool on it and there's more than that required.

``` 
flatpak run org.freedesktop.appstream-glib validate com.dhsdevelopments.Climaxima.appdata.xml
com.dhsdevelopments.Climaxima.appdata.xml: FAILED:
• tag-invalid           : <metadata_license> is not valid [MIT]
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
• tag-missing           : <release> required
Validation of files failed
```

Reply:
Huh, and now I see I was reviewing an older commit! Looks good now. I'd add a project_licence tag as well fwiw